src/plugins/data/public/ui/filter_bar/_global_filter_item.scss

```diff
@@ -33,14 +33,13 @@
 }
 
 .globalFilterItem-isError, .globalFilterItem-isWarning {
-  text-decoration: none;
-
   .globalFilterLabel__value {
     font-weight: $euiFontWeightBold;
   }
 }
 
 .globalFilterItem-isError {
+  text-decoration: none;
```

Comment:
Should disabled **error** filters also get the strikethrough?

Reply:
I think it would make sense and I'd like to add a strikethrough. 
I just was looking at the pr that introduced this error state and it seemed that it was intentional to **not** add a strike through for error filter. 

https://github.com/elastic/kibana/pull/59298/commits/75451fb4810be884b0d8ef733924cf57df5b2845#diff-53281e43bd82ecd67995caaec78ffea3R36

Could you check? maybe you would recall that decision?

Reply:
Yeah, lets just allow the strikethrough for all disabled pills no matter the error/warning states.

Reply:
@cchaos, updated